app.js

```diff
@@ -0,0 +1,160 @@
+let myLibrary = [];
+
+function setMyLibrary() {
+  if (localStorage.getItem("myLibrary") !== null) {
+    myLibrary = JSON.parse(localStorage.getItem("myLibrary"));
+    loadBooks();
+  } else {
+    myLibrary = [];
+  }
+}
+
+const booksContainer = document.querySelector(".books-container");
+const bookTitleInput = document.querySelector("#book-title");
+const bookDescriptionInput = document.querySelector("#book-description");
+const pagesInput = document.querySelector("#pages-number");
+const readStatusInput = document.querySelector("#read-status");
+const buttonInput = document.querySelector(".btn");
+const btnAddNewBook = document.querySelector(".btn-add-new-book");
+const newBookForm = document.querySelector(".new-book");
+
+buttonInput.addEventListener("click", createNewBook);
+
+function createNewBook() {
+  let title = bookTitleInput.value;
+  let desc = bookDescriptionInput.value;
+  let pages = pagesInput.value;
+  let status = readStatusInput.checked;
+
+  let book = {title, desc, pages, status};
```

Comment:
- [ ] [REQUIRED] Book should have a constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library)).